src/explore-education-statistics-admin/src/services/keyStatisticService.ts

```diff
@@ -12,12 +12,28 @@ export interface KeyStatisticDataBlockCreateRequest {
   guidanceText?: string;
 }
 
+export interface KeyStatisticTextCreateRequest {
+  title: string;
+  statistic: string;
+  trend?: string;
+  guidanceTitle?: string;
+  guidanceText?: string;
+}
+
 export interface KeyStatisticDataBlockUpdateRequest {
   trend?: string;
   guidanceTitle?: string;
   guidanceText?: string;
 }
 
+export interface KeyStatisticTextUpdateRequest {
```

Comment:
On the backend, we've been making each their own object - this means that responses and requests have to be deliberately altered. I think this more to prevent mischief on the response side - i.e. someone makes a change that unwittingly means an endpoint suddenly starts returning something we didn't intend. 

Not sure there is as much danger on the request side - someone could pass in something we didn't intend, but then it wouldn't get used, I think. Maybe there is some danger in some edge case.

I'm inclined to keep them separate - if only because that's how they are in the backend. But if others think differently, happy to go along with something else.

Reply:
In the future, you could just omit properties that don't belong in the update request by extending from the create request. My only gripe with you doing it this way is you're repeating yourself in two interfaces & will have to update both if anything about both change

Reply:
I think across codebase you'll be able to find examples of us duplicating models or not, depending on the scenario. Some cases the update will differ from the create.

However, I probably would suggest that we could create single request view model i.e. `KeyStatisticTextSaveRequest`, then inherit from it with the create/update requests i.e.

```cs
public record KeyStatisticTextSaveRequest {
  // the fields...
}

public record KeyStatisticTextCreateRequest : KeyStatisticTextSaveRequest;

public record KeyStatisticTextUpdateRequest : KeyStatisticTextSaveRequest;
```

Typically, with REST APIs, you expect a bit of symmetry when creating/updating i.e. anything you can set in the create can also be done in the update (if it's a PUT).

Of course, this isn't always the case, but I don't think key statistics will really warrant any major discrepancies in the request models right now. We can always refactor it anyway!

If we do make the above change, I'd go with what Luke has suggested and model this in a similar way in the frontend too.

Reply:
I've introduced `KeyStatisticTextSaveRequest` on both the backend and frontend.